pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java

```diff
@@ -319,13 +321,13 @@ private void testDistinctInnerSegmentHelper(String[] queries, boolean isPql)
   @Test
   public void testDistinctInnerSegment()
       throws Exception {
-    testDistinctInnerSegmentHelper(new String[]{
-        "SELECT DISTINCT(intColumn, longColumn, floatColumn, doubleColumn, stringColumn, bytesColumn) FROM testTable LIMIT 10000",
-        "SELECT DISTINCT(stringColumn, bytesColumn, floatColumn) FROM testTable WHERE intColumn >= 60 LIMIT 10000",
-        "SELECT DISTINCT(intColumn, bytesColumn) FROM testTable ORDER BY bytesColumn LIMIT 5",
-        "SELECT DISTINCT(ADD ( intColumn,  floatColumn  ), stringColumn) FROM testTable WHERE longColumn < 60 ORDER BY stringColumn DESC, ADD(intColumn, floatColumn) ASC LIMIT 10",
-        "SELECT DISTINCT(floatColumn, longColumn) FROM testTable WHERE stringColumn = 'a' ORDER BY longColumn LIMIT 10"
-    }, true);
+    testDistinctInnerSegmentHelper(
```

Comment:
I feel the original formatting is better. You can skip the reformatting by adding `//@formatter:off`, see `AggregationFunctionUtils.isFitForDictionaryBasedComputation()` for details.